src/ts/config/webpack.config.js

```diff
@@ -574,8 +577,8 @@ module.exports = function (webpackEnv) {
       // to restart the development server for webpack to discover it. This plugin
       // makes the discovery automatic so you don't have to restart.
       // See https://github.com/facebook/create-react-app/issues/186
-      isEnvDevelopment &&
-        new WatchMissingNodeModulesPlugin(paths.appNodeModules),
+      // isEnvDevelopment &&
+      //   new WatchMissingNodeModulesPlugin(paths.appNodeModules),
```

Comment:
Did you intend to comment this out?

Reply:
Yes, it was crashing watching my `node_modules` folder. I'm not sure if I should remove it entirely

Reply:
I don't see this crash.  Do you have a trace?

Also, it looks like you added the `watchOptions` directive above.  Does that avoid the problem?